Client/Frontend/Login Management/LoginDataSource.swift

```diff
@@ -52,7 +52,7 @@ class LoginDataSource: NSObject, UITableViewDataSource {
             }
 
             // Fade in the cell while dismissing the search or the cell showing suddenly looks janky
-            if viewModel.isDuringSearchControllerDismiss {
+            if viewModel.isDuringSearchControllerDismiss && !hideSettings {
```

Comment:
Since there's a `if hideSettings {` up here, we could use that to get an else if where we would be in the case of `!hideSettings`. So something like this:
```
            if hideSettings {
                cell.isHidden = true

            // Fade in the cell while dismissing the search or the cell showing suddenly looks janky
            } else if viewModel.isDuringSearchControllerDismiss {
                cell.isHidden = false
                cell.contentView.alpha = 0
                cell.accessoryView?.alpha = 0
```

Reply:
Fixes made as requested. Thanks!